test/fixtures/comparison-files/lib/actions/ConfigAction.test.ts/override-configurations/StubEngine2_forConfigWithRelativePathScenario.yml.goldfile

```diff
@@ -7,6 +7,6 @@
     disable_engine: false
 
     # Some description for top_field
-    top_field: # Modified from: null
+    top_field: # Modified from: {}
```

Comment:
Added in a more appropriate default for this field - which is why the goldfile changed.